ui/page/send/layout.go

```diff
@@ -90,6 +84,57 @@ func (pg *Page) topNav(gtx layout.Context) layout.Dimensions {
 	)
 }
 
+func (pg *Page) getMoreItem() []moreItem {
+	return []moreItem{
+		{
+			text:   "Advanced mode",
+			button: new(widget.Clickable),
+			id:     UTXOPageID,
+			action: func() {
+				pg.ChangeFragment(NewUTXOPage(pg.Load))
+			},
+		},
+		{
+			text:   "Clear all fields",
+			button: new(widget.Clickable),
+			action: func() {
+				pg.resetFields()
+				pg.moreOptionIsOpen = false
+			},
+		},
+	}
+}
+
+func (pg *Page) layoutOptionsMenu(gtx layout.Context) {
+	inset := layout.Inset{
+		Top:  unit.Dp(30),
+		Left: unit.Dp(-120),
```

Comment:
```suggestion
		Top: values.MarginPadding30,
		Left: values.MarginPaddingMinus120,
```
Add them to the Dimensions.go file if they dont exist.